README.md

```diff
@@ -1 +1,34 @@
-# mento-tooling
\ No newline at end of file
+# Mento Core TS
+
+This repo contains the source code of the Mento core contracts type generation and NPM package deployment.
```

Comment:
This should be written as if the audience is somebody looking to interact with the mento protocol via the generated types and factories. I understand why you phrased it like this because that's how we've been talking about it internally but from the perspective of somebody external that's weird because all repos with js/ts libraries do NPM package deployment.

Let's phrase more around the lines of "this package provides auto-generated typescript types and ethers.js factories for interacting with the Mento Protocol"

Reply:
yeah, I made the mistake of writing the readme for the repo.